### ISSUE TITLE
Consider multiple lessons in a row of the same class as only one lesson

### ISSUE THREAD

**Issue by Iftch** [arminghofrani/chalkboard#3]
For instance, if I have two lessons of mathematics in a row (even if there are a break between theme), when I add new event for mathematics and I click on the "Due specific class" to choose the time, those two lessons will be considered as only one lesson.

**Comment by arminghofrani** [comment]
Yes, I've thought about doing this, but the problem is differentiating between cases where the two lessons should be considered as one lesson, and cases where they should be considered as two separate lessons.

One way of doing this is by looking at the time between the two lessons. For example, if the break between the two lessons is shorter than an hour, the two lessons will be considered as one, but if the break is longer, they will be considered separately. What I don't like about this approach is that the threshold (in this case one hour) would have to be chosen completely arbitrarily, which I'd rather avoid.

Differentiating between the two cases based on factors other than the time between the two lessons also doesn't make much sense to me. However, if you have another idea please let me know, and I'd be glad to implement it.

Until then, if you're in the situation you described in your comment, just set the event as due on the first of the two lessons - this means you'll get the reminders at the right time, and the event would be displayed correctly in the 'Overview' screen.
